app/src/androidTest/java/com/example/triptracker/itinerary/MockItineraryList.kt

```diff
@@ -1,6 +1,5 @@
 package com.example.triptracker.itinerary
 
-import android.net.Uri
```

Comment:
Clever design choice going from URI to just Strings, it simplify things and improve code readability as well